### DISCUSSION TITLE
Necesito que me ayudes con mi cuenta de github , no puedo acceder

### DISCUSSION THREAD

**Discussion by LeaaBertero** [community/community#132924] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

no puedo acceder a mi cuenta de github, pido por favor que me ayuden, perdi el codigo de factor de autenticacion y no puedo acceder, , necesito ayuda por favor, para que pueda acceder. gracias !

**Comment by davevad93** [comment]
Hi @LeaaBertero ,

If you have lost your 2FA credentials, you should use your recovery codes to gain access to your account, but if you have also lost your recovery codes, you are permanently locked out of your account. However, you can unlink the email address associated with your locked account and create a new account using that email. For more information take a look at this (here's the link to the Spanish version of the docs) : 

- [Unlinking your email address from a locked account](https://docs.github.com/es/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/unlinking-your-email-address-from-a-locked-account)

**Comment by ebndev** [comment]
Hi @LeaaBertero, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this post for now, but please feel encouraged to create a new discussion in English if you still need support / the answer above did not fully resolve your issue.
